pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -189,3 +198,68 @@ func outputHelper(info, namespace string, singleNamespace bool) string {
 	}
 	return fmt.Sprintf("<%s %s>", namespace, info)
 }
+
+type byID []graph.Node
```

Comment:
We already have this: https://github.com/openshift/origin/blob/4bc1af7d29eb1321fcb7fe25911c7935a5f526b4/pkg/api/graph/graph.go#L174